pkg/storaged/dialog.jsx

```diff
@@ -1161,13 +1163,15 @@ export function init_active_usage_processes(client, usage) {
 }
 
 export const StopProcessesMessage = ({ mount_point, users }) => {
+    const mock = useMock()?.busy;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20230302-101354-40692ee9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)